scripts/update_feedstocks_submodules.py

```diff
@@ -40,8 +42,16 @@
             feedstocks_repo.submodules[feedstock.package_name].remove()
 
         # Add the new submodule.
-        feedstocks_repo.create_submodule(feedstock.package_name, repo_subdir,
-                                         url=feedstock.clone_url)
+        try:
+            feedstocks_repo.create_submodule(feedstock.package_name, repo_subdir,
+                                             url=feedstock.clone_url)
+        except ValueError:
+            warning.warn(
+                    "Unable to add the submodule {}. "
+                    "This is likely because the repo has no commits, "
+                    "which likely means something went wrong with feedstock generation. "
+                    "Will skip adding this submodule and continue.".format(feedstock.package_name)
+            )
```

Comment:
I don't thinks so because if it raises an error we can't process the rest of the working feedstocks.

In this case, I am looking at yt, but there will be more until we can get the issue with `libmo_unpack` fixed. Also, I doubt this is the last time that we are going to want to bypass broken feedstocks.

The main problem is less that they get added to the feedstocks repo and more that permissions to feedstocks aren't being handed out to people in a timely manner when these issues occur.